### PR TITLE
Fix a bug  in example/fib/app.go line:69

### LINKED ISSUE #2862: The Fibonacci example exits unexpectedly on windows.
### Description

When running the Fibonacci example under windows after providing the first number the program exits with an error `unexpected newline`
```
PS C:\Users\Aaron\projects\opentelemetry-go\example\fib> go run .
What Fibonacci number would you like to know:
8
Fibonacci(8) = 21
What Fibonacci number would you like to know:
unexpected newline
exit status 1
```

### Environment

- OS: windows Powershell (does not effect WSL, git bash, linux powershell)
- Architecture: x86-64
- Go Version: 
    - go version go1.16 windows/amd64
    - go version go1.18.1 windows/amd64
- opentelemetry-go version: main, 68d538a24f998a

### Steps To Reproduce

1. Use the configuration running in powershell
2. Run:
```
PS C:\Users\Aaron\projects\opentelemetry-go> cd .\example\fib
PS C:\Users\Aaron\projects\opentelemetry-go\example\fib> go run .
What Fibonacci number would you like to know:
8
Fibonacci(8) = 21
What Fibonacci number would you like to know:
unexpected newline
exit status 1
```
cd fib
4. See error: `unexpected newline`

### Expected behavior

This should operate the same as linux, wsl, or git bash.  Where it will prompt you for another number until you use ctrl+c.

### PR DESCRIPTION
**Pull request by Frapschen**
Fixes #2862

## dev env:
* system:win10
* IDE:goland

## Describe：

when I run the example of fib,the loop will be Breaked by the Fscanf(). 

the error message:
```
What Fibonacci number would you like to know:
10
Fibonacci(10) = 55
What Fibonacci number would you like to know: 
unexpected newline

Process finished with the exit code 1

```

## Fix method：

append a "\n" to the param string in  Fscanf().

### CODE REVIEW

**Comment by dmathieu** [comment]
See #2717.

**Comment by Frapschen** [comment]
ok, I have modified the "getting-started.md" file, which refers to #2644.

File: CHANGELOG.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Delegated instruments are unwrapped before delegating Callbacks. (#2784)
 - Resolve supply-chain failure for the markdown-link-checker GitHub action by calling the CLI directly. (#2834)
 - Remove import of `testing` package in non-tests builds. (#2786)
+- modify the Fscanf function param in example/fib/app.go line:69 to avoid process break by "unexpected newline". (#2860)
```
Comment (pellared) [inline CHANGELOG.md:26]: TBH I do not think it is needed (or even worth mentioning)

**Review by pellared — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (Frapschen) [reply CHANGELOG.md]: Ok，I just follow the contribution guidelines.

**Comment by MadVikingGod** [comment]
After testing this bug does affect windows powershell, see #2862.

I believe that this change will fix powershell and not adversely affect any other platform.  If someone has the chance to test with a mac I would appreciate it.

**Comment by MadVikingGod** [comment]
I would also like to point out that the reason this PR should be considered while the first two PRs weren't is this description has enough information to reproduce.  

We welcome both bug reports and PRs, but we need to know why a change should be made.  If you start with a bug report before or at the same time as a PR then we can get the reasoning behind the change.

**Review by MrAlias — APPROVED** [review]

**Comment by MrAlias** [comment]
@MadVikingGod are you able to verify this PR resolves #2862? I have verified this does not change the working behavior on a Linux system.

**Review by MadVikingGod — APPROVED** [review]
Yes, this does fix the powershell, and it doesn't affect mac or windows linuxs

**Pull request merged**